src/main/java/gyro/aws/elasticsearch/ElasticsearchAdvancedSecurityOptions.java

```diff
@@ -60,7 +60,7 @@ public void setEnableInternalUserDatabase(Boolean enableInternalUserDatabase) {
     }
 
     /**
-     * Credentials for the master user.
+     * Configure the master user if advanced security is enabled.
```

Comment:
Can you change all the docs similar to this
 - If its a subresource <Space separated type> configuration.
 - If condition is present, use configuration name (``-`` separated name) and provide the conditional value

```suggestion
     * Master user option configuration.  Can only be set when ``enable-advance-security-option`` is set to ``true``.
```